src/main/java/seedu/address/model/person/predicates/HideAppointmentPredicate.java

```diff
@@ -51,7 +51,8 @@ public boolean test(Appointment appt) {
             break;
         case IS_MARKED:
             String kw = keywords.get(0);
-            passed = kw.equals("marked") || kw.equals("m") ? appt.isMarked() : !appt.isMarked();
+            passed = kw.equalsIgnoreCase("marked") || kw.equalsIgnoreCase("m")
+                    ? appt.isMarked() : !appt.isMarked();
```

Comment:
Would it be better to separate the `?` and `:` segments into 2 separate lines for easier readability?